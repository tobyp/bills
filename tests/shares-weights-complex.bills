#!./bills
A	0.5B(CD)2(EFG)3(H(IJ)K)4(2L3M)5(N(OP)2(QR)(ST5U6(V7W)))(X(YZ)).	12870.0
