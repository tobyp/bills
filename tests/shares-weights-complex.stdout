todo: B pays 390.0 to A
todo: C pays 390.0 to A
todo: D pays 390.0 to A
todo: E pays 520.0 to A
todo: F pays 520.0 to A
todo: G pays 520.0 to A
todo: H pays 780.0 to A
todo: I pays 390.0 to A
todo: J pays 390.0 to A
todo: K pays 780.0 to A
todo: L pays 1248.0 to A
todo: M pays 1872.0 to A
todo: N pays 780.0 to A
todo: O pays 390.0 to A
todo: P pays 390.0 to A
todo: Q pays 780.0 to A
todo: R pays 780.0 to A
todo: S pays 60.0 to A
todo: T pays 60.0 to A
todo: U pays 300.0 to A
todo: V pays 45.0 to A
todo: W pays 315.0 to A
todo: X pays 390.0 to A
todo: Y pays 195.0 to A
todo: Z pays 195.0 to A
